internal/bminventory/inventory.go

```diff
@@ -2700,7 +2699,7 @@ func shouldHandle(params installer.V2PostStepReplyParams) bool {
 
 func (b *bareMetalInventory) handleReplyError(params installer.V2PostStepReplyParams, ctx context.Context, log logrus.FieldLogger, h *models.Host, exitCode int64) error {
 	if exitCode == MediaDisconnected {
-		if err := b.handleMediaDisconnection(params, ctx, log, h); err != nil {
+		if err := b.hostApi.HandleMediaDisconnected(ctx, h); err != nil {
```

Comment:
Should we add a check `if host.MediaStatus != nil && *host.MediaStatus != models.HostMediaStatusDisconnected` ?

Reply:
we have this check inside HandleMediaDisconnected